linux/iscsi.go

```diff
@@ -57,7 +57,7 @@ const (
 
 var (
 	iscsiMutex           sync.Mutex
-	targetVendorPatterns = []string{"com.nimblestorage", "com.3pardata"}
+	targetVendorPatterns = []string{"com.nimblestorage", "com.3pardata", "org.truenas.ctl"}
```

Comment:
We can look into removing these altogether. As filtering is not required and we can always rely on full target IQN name and serial provided during node stage.